pipeline/src/pipeline.rs

```diff
@@ -383,16 +383,6 @@ impl<T: FieldElement> Pipeline<T> {
         }
     }
 
-    pub fn from_pil_with_evaluated_fixed_cols(
```

Comment:
I think this function might still be useful for getting verification keys. But yea let's remove it here and if I need it again I'll bring it back.